core/tests/protractor/embedding.js

```diff
@@ -161,33 +224,46 @@ describe('Embedding', function() {
       driver.get(
         general.SERVER_URL_PREFIX + general.SCRIPTS_URL_SLICE +
         'embedding_tests_dev_i18n_0.0.1.html');
-      browser.switchTo().frame(driver.findElement(by.xpath(
-          "//div[@class='protractor-test-embedded-exploration']/iframe")));
+
+      driver.findElement(
+        by.css('#embedding-exploration-id')).sendKeys(explorationId);
```

Comment:
Can you actually use CSS classes instead of IDs in the HTML? That's more in line with our usual Protractor convention. Also prefix those classes with protractor-test-*

Ditto below.

Reply:
have changed to use classes.